FordLabs/retroquest
# Jest

## Overview
Change unit testing framework from Jasmine to Jest.  Save 75% of time executing unit tests

Connects #274 

## Testing Instructions
Run the following commands and observe results
```
yarn unit
```
```
yarn unit-coverage
```
```
yarn unit-watch
```

## Note
4 of the test suites had no tests.  These were deleted.
